_posts/2025-02-01-hello-debezium-community.adoc

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  Hello Debezium Community!
+date:   2025-02-01 12:00:00 +0100
+tags: [ community, news ]
+author: alvarvg
+---
+
+Hello Debezium Community! 
+
+I am Alvar Viana and exactly one month ago I joined Red Hat and the Debezium team. I am a software engineer with almost 10 years of professional experience.
+In this years, I have always worked for product companies/proyects. Started as backend/embedded developer for a company that was acquired ABB.
+A few years later I changed my life, changed cities, changed jobs and changed roles. Since them, I have been working as a Quality Engineer, at adidas and Sysdig. 
+As of today, I am happily joining Red Hat and debezium.  
+
+My contact with red hat has been little, although since I was a child I have looked at the company with admiration, since half a year ago I have been in different selection processes.
```

Comment:
```suggestion
I've had limited contact with Red Hat, although since I was a child, I have admired the company; since half a year ago, I have been in different selection processes.
```